addons/http_routing/models/ir_http.py

```diff
@@ -297,7 +297,7 @@ def get_frontend_session_info(self):
         modules = IrHttpModel.get_translation_frontend_modules()
         user_context = request.session.context if request.session.uid else {}
         lang = user_context.get('lang')
-        translation_hash = request.env['ir.translation'].get_web_translations_hash(modules, lang)
+        translation_hash = request.env['ir.translation.code'].get_web_translations_hash(modules, lang)
```

Comment:
Yes, I did this because the old translations for web is stored in ORM cache.
I am not sure about the overhead of abstractObject and ORM cache(is ormcache shared by workers?).
I will make a version to store it directly in memory.

Reply:
The overhead is not big, it just feels cleaner to avoid an object for not persistency needs.